### PR TITLE
feat(ldap): Add attribute studentIdentifiers to LDAP schema

### PR DESCRIPTION
**Pull request by cuadradek**
- Attribute studentIdentifiers was added to perun-schema.ldif.

### CODE REVIEW

File: perun-utils/ldapc-scripts/schemas/perun-schema.ldif [hunk 1]
```diff
@@ -72,6 +72,7 @@ olcAttributeTypes: {67}( 1.3.6.1.4.1.8057.2.80.73 NAME 'uuid' DESC 'UUID of enti
 olcAttributeTypes: {68}( 1.3.6.1.4.1.8057.2.80.74 NAME 'rpEnsureVoDefinition' DESC 'Short name of VO, which proxy check if the user is member of' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
 olcAttributeTypes: {69}( 1.3.6.1.4.1.8057.2.80.75 NAME 'rpEnableEnsureVoFiltering' DESC 'If set to True, proxy will trigger the ensureVo filter' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
 olcAttributeTypes: {70}( 1.3.6.1.4.1.8057.2.80.76 NAME 'rpLoginURL' DESC 'URL where user can log in to the service.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {71}( 1.3.6.1.4.1.8057.2.80.77 NAME 'studentIdentifiers' DESC 'All student identifiers generated by Perun' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```
Comment (balcirakpeter) [inline perun-utils/ldapc-scripts/schemas/perun-schema.ldif:75]: Please set Equality, Ordering, etc. as it is for schacPersonalUniqueCodes.

File: perun-utils/ldapc-scripts/schemas/perun-schema.ldif [hunk 2]
```diff
@@ -72,6 +72,7 @@ olcAttributeTypes: {67}( 1.3.6.1.4.1.8057.2.80.73 NAME 'uuid' DESC 'UUID of enti
...
 -
 replace: olcObjectClasses
 olcObjectClasses: {0}( 1.3.6.1.4.1.8057.2.80.4 NAME 'perunUser' DESC 'User managed by Perun' SUP inetOrgPerson STRUCTURAL MUST ( perunUserId $ isServiceUser $ isSponsoredUser ) MAY ( preferredMail $ userCertificateSubject $ uidNumber $ login $ eduPersonPrincipalNames $ userPassword $ memberOfPerunVo $ libraryIDs $ schacHomeOrganizations $ eduPersonScopedAffiliations $ bonaFideStatus $ groupNames $ institutionsCountries $ isCesnetEligible $ loa $ internalUserIdentifiers $ eduPersonOrcid $ loaFenix $ adminOfVo $ adminOfGroup $ adminOfFacility $ eduPersonEntitlement $ europeanStudentID $ eIDASPersonIdentifier $ timezone $ phone $ address $ aups $ tcsMails $ sshPublicKey $ sponsoredMembershipInOrganizations $ userIdentities $ schacPersonalUniqueCodes $ securityImage $ mfaEnforced $ mfaTokens $ uuid ) )
```
Comment (balcirakpeter) [inline perun-utils/ldapc-scripts/schemas/perun-schema.ldif:78]: New attribute has to be put also to the list of user attributes.

**Review by balcirakpeter — CHANGES_REQUESTED** [review]

File: perun-utils/ldapc-scripts/schemas/perun-schema.ldif (hunk 1 above)
Comment (cuadradek) [reply perun-utils/ldapc-scripts/schemas/perun-schema.ldif]: Done.

File: perun-utils/ldapc-scripts/schemas/perun-schema.ldif (hunk 2 above)
Comment (cuadradek) [reply perun-utils/ldapc-scripts/schemas/perun-schema.ldif]: Done.

**Review by balcirakpeter — APPROVED** [review]

**Pull request merged**